### PR TITLE
more brein damage line

### PR DESCRIPTION
**Pull request by LetterN**
adds more brain damage line
[Common] Astraeus states, "Reeee its a ship not a station get it right"

### CODE REVIEW

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -121,4 +123,15 @@
         "sheadow lings",
         "abdoocters"
     ]
+	"game":[
+		"tEWTeEN",
+		"STAtIO TweOTWEEN",
+	]
+	"ai":[
```
Comment (MrMaclang) [inline strings/brain_damage_lines.json]: more ai names needed

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -49,15 +49,17 @@
...
         "Tajaran has warrrres, if you have coin",
         "I'VE GOT BALLS OF STEEL",
         "NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
         "i will snatch erry motherfucker birthday",
         "the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
-        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?"
+        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
+		"EFTEL @pick(game) TwewTEEN!",
+		"@pick(ai) RoUEG! CAll ShuTEEL!"
```
Comment (ninjanomnom) [inline strings/brain_damage_lines.json]: This file is space indented, not tab

**Comment by ninjanomnom** [comment]
All these lines are sub par and fixing the station/ship brain damage line seems pointless in this particular case.

**Comment by MrMaclang** [comment]
Fixed, Now i hate tabs

File: strings/brain_damage_lines.json [hunk 3]
```diff
@@ -121,4 +123,17 @@
         "sheadow lings",
         "abdoocters"
     ]
+	"game":[
```
Comment (ninjanomnom) [inline strings/brain_damage_lines.json]: not fixed, still tabs

Comment (MrMaclang) [reply strings/brain_damage_lines.json]: gah, fixing

File: strings/brain_damage_lines.json [hunk 4]
```diff
@@ -121,4 +123,17 @@
         "sheadow lings",
         "abdoocters"
     ]
+    "game":[
+		" ",
```
Comment (ninjanomnom) [inline strings/brain_damage_lines.json]: still tabs

do you not have a way to view tabs and spaces in your editor?

Comment (MrMaclang) [reply strings/brain_damage_lines.json]: properly fixed it

**Comment by Vivalas** [comment]
This is not exactly quality content. Brain damage lines tend to be bad in general, but these are somewhat worse, in my opinion, and they have even less chance of being used.

Now, I understand it's a meme, but at least make a quality meme. Closed, unless another staff member vouches for these.

**Pull request closed without merging**